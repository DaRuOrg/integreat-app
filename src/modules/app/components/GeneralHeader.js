import React from 'react'
import locationIcon from '../assets/location-icon.svg'
import Header from 'modules/layout/components/Header'

class GeneralHeader extends React.Component {
  render () {
    return <Header actionItems={[{href: '/', iconSrc: locationIcon}]} />
  }
}

export default GeneralHeader
