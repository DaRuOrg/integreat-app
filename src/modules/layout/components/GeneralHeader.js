import React from 'react'
import locationIcon from '../assets/location-icon.svg'
import Header from '../containers/Header'
import HeaderActionItem from '../HeaderActionItem'

class GeneralHeader extends React.Component {
  render () {
    return <Header actionItems={[new HeaderActionItem({href: '/', iconSrc: locationIcon})]} />
  }
}

export default GeneralHeader
