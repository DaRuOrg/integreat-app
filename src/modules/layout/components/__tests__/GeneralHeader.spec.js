import { shallow } from 'enzyme'
import React from 'react'
import GeneralHeader from '../GeneralHeader'

describe('GeneralHeader', () => {
  test('should match snapshot', () => {
    const component = shallow(<GeneralHeader />)
    expect(component).toMatchSnapshot()
  })
})
