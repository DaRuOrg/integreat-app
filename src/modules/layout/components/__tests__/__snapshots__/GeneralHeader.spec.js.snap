// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneralHeader should match snapshot 1`] = `
<Header
  actionItems={
    Array [
      HeaderActionItem {
        "_dropDownElement": undefined,
        "_href": "/",
        "_iconSrc": "test-file-stub",
      },
    ]
  }
  navigationItems={Array []}
/>
`;
